GetStream/stream-chat-android
# [3378] Create a dedicated environment for UI tests

### 🎯 Goal

We need a separate testing environment for UI tests to avoid flakiness in automated and manual UI tests.

### 🎨 UI Changes

<img src="https://user-images.githubusercontent.com/9600921/165359959-dd365baf-c915-438c-823f-fff3896f892d.jpeg" width="40%">

### 🧪 Testing

1. Run `stream-chat-android-ui-uitests` sample app
2. Ensure that you are to login with provided users

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
